Cargo.toml

```
@@ -7,7 +7,7 @@ members = ["cli", "lib", "lib/gen-protos", "lib/proc-macros", "lib/testutils"]
 [workspace.package]
 version = "0.27.0"
 license = "Apache-2.0"
-rust-version = "1.84" # NOTE: remember to update CI, contributing.md, changelog.md, and install-and-setup.md
+rust-version = "1.84" # NOTE: remember to update mise.toml, CI, contributing.md, changelog.md, and install-and-setup.md
```

```suggestion
rust-version = "1.84" # NOTE: remember to update CI, mise.toml contributing.md, changelog.md, and install-and-setup.md
```

nit: I feel like "CI" should go first since it's most important, in some sense.